.github/styles/Vocab/Docs/accept.txt

```diff
@@ -4,6 +4,8 @@ boolean
 [Bb]usiness
 datasource
 datastore
+defaultdb
```

Comment:
Where did we need to add this? Spelling check shouldn't hit things with the `literal` markup